dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java

```diff
@@ -0,0 +1,65 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+
+/**
+ * Task used to populate fields: owner, create_date, asset_subtype of the identifier table
+ */
+public class Task201014UpdateColumnsValuesInIdentifierTable implements StartupTask {
+
+    protected static final String UPDATE_CONTENTLETS =
+            "update identifier set owner=q.i_owner, create_date=q.create_date, asset_subtype=q.asset_subtype from"
+            + "(select cont.identifier, cont.mod_user i_owner, inode.idate create_date, "
+            + "struc.velocity_var_name asset_subtype from contentlet cont, "
+            + "inode inode, structure struc "
+            + "where inode.inode=cont.inode "
+            + "and cont.structure_inode = struc.inode) q where id=q.identifier";
+
+    private static final String UPDATE_FOLDER = "update identifier set owner=q.i_owner, create_date=q.create_date from "
```

Comment:
Codacy found an issue: [The String literal "update identifier set owner=q.i_owner, create_date=q.create_date from " appears 4 times in this file; the first occurrence is on line 21](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6327767)